avocado/plugins/plugins.py

```diff
@@ -46,7 +46,9 @@ def run(self, args):
             (dispatcher.CLIDispatcher(),
              'Plugins that add new options to commands (avocado.plugins.cli):'),
             (dispatcher.JobPrePostDispatcher(),
-             'Plugins that run before/after the execution of jobs (avocado.plugins.job.prepost):')
+             'Plugins that run before/after the execution of jobs (avocado.plugins.job.prepost):'),
+            (dispatcher.ResultDispatcher(),
+             'Plugins that generate job result in different formats (avocado.plugins.result):')
```

Comment:
Sure, it's good description for now, we can expand it when you introduce the other (the one which allows calls during the execution) type.